src/styles/_header.scss

```diff
@@ -37,5 +36,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms linear;
```

Comment:
You should specify transition properties directly
Default is all, but it could affect old browsers' performance